### PR TITLE
Error count template

### PR DESCRIPTION
**Pull request by kforsberg**

### CODE REVIEW

File: templates/private-sql-data-monitor.json [hunk 1]
```diff
@@ -82,6 +82,13 @@
       "metadata": {
         "description": "How many seconds to wait between runs"
       }
+    },
+    "errorCountRows": {
+      "type": "int",
+      "defaultValue": 0,
+      "metadata": {
+        "description": "How many rows returned will trigger an error"
```
Comment (lukejgaskell) [inline templates/private-sql-data-monitor.json]: technically it's how many rows is not error

**Review by sam2k13 — APPROVED** [review]

File: templates/private-sql-data-monitor.json (hunk 1 above)
Comment (lukejgaskell) [reply templates/private-sql-data-monitor.json]: maybe something like "Greater than this many rows will be considered an error"

maybe something less wordy if you can think of it lol

File: templates/sql-data-monitor.json [hunk 2]
```diff
@@ -64,6 +64,13 @@
       "metadata": {
         "description": "How many seconds to wait between runs"
       }
+    },
+    "errorCountRows": {
+      "type": "int",
+      "defaultValue": 0,
+      "metadata": {
+        "description": "How many rows returned will trigger an error"
```
Comment (lukejgaskell) [inline templates/sql-data-monitor.json]: same as above

File: templates/private-sql-data-monitor.json [hunk 3]
```diff
@@ -82,6 +82,13 @@
       "metadata": {
```
Comment (lukejgaskell) [inline templates/private-sql-data-monitor.json:82]: Probably need the readme updated too if you didn't in the last PR

**Review by lukejgaskell — CHANGES_REQUESTED** [review]

**Pull request closed without merging**